packages/react-query/src/scripts/cli.ts

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env node
+'use strict'
+
+import { Command } from 'commander'
```

Comment:
https://github.com/coauthors/coauthors/blob/main/packages/coauthors/package.json#L41 This support TypeScript type could you check this?

Reply:
> if you want to support @commander-js/extra-typings in next Pull Request, In my opinion, it's also good!

I will apply it right away!